directory.json

```diff
@@ -1,6 +1,7 @@
 {
+  "Freemesh Denmark": "https://raw.githubusercontent.com/Freemesh-Denmark/freemesh-api/master/FreemeshDenmark-api.json",
```

Comment:
If you refer to this as a key, then the explanation was not complete.

I will change this to "freemesh-denmark" then